packages/components/datepicker/src/components/osds-datepicker/interfaces/attributes.ts

```diff
@@ -0,0 +1,30 @@
+interface OdsDatepickerAttribute {
+  /**
+   * Defines if the Datepicker should be clearable or not (displays a clear button)
+   */
+  clearable?: boolean;
+  /**
+   * Defines if the Datepicker should be disabled or not (lower opacity and not interactable)
+   */
+  disabled?: boolean;
+  /**
+   * Defines if the Datepicker should display an error message
+   */
+  error?: string;
```

Comment:
why a string?
error message are managed by form field, this component should behave like an input, thus error only serves as a style toggle